core/schemas/package.py

```diff
@@ -0,0 +1,276 @@
+import json
+from datetime import datetime, timezone
+from typing import Any, ClassVar, Dict, List, Optional, Tuple
+
+from pydantic import BaseModel, Field, model_validator
+from typing_extensions import Self
+
+from core.schemas import entity, indicator, observable
+from core.schemas.observable import ObservableTypes
+
+
+class YetiPackageRelationship(BaseModel):
+    target: str
+    link_type: str = "observes"
+
+
+class YetiPackage(BaseModel):
```

Comment:
It would be good to add a docstring here detailing what th attributes represent (eg. I have no idea what "source" is supposed to be here)

Reply:
Source is mainly used for context but I agree, docstring would help :)

Reply:
Fixed in 5b682a6b1a7bde980f127a5fcd8cc6bb46a47d3f